Azure/azure-cli
# [Storage] `az storage entity`: Support specifying `EdmType` for `--entity`

**Description**

Support `EdnType` for storage entity to fix #22018 

**Testing Guide**

```
az storage entity insert -e PartitionKey=AAA RowKey=BBBB SizeInBytes=86222504922 SizeInBytes@odata.type=Edm.Int64 binaryProperty=AAECAwQF binaryProperty@odata.type=Edm.Binary -t ystable --account-name yssa --auth-mode login
```
```
az storage entity show --partition-key AAA --row-key BBBB -t ystable --account-name yssa --auth-mode login
```
 


---

This checklist is used to make sure that common guidelines for a pull request are followed.
